### PR TITLE
[issue-264] fix any_holidays_during_work_week? method to use expected logic

### PR DESCRIPTION
**Pull request by ppeble**
This addresses https://github.com/holidays/holidays/issues/264.

### CODE REVIEW

**Comment by ppeble** [comment]
@ttwo32 I took this opportunity to update our README for clarity. Please give it a look and let me know what you think. 🙇

File: README.md [hunk 1]
```diff
@@ -1,182 +1,317 @@
 # Ruby Holidays Gem [![Build Status](https://travis-ci.org/holidays/holidays.svg?branch=master)](https://travis-ci.org/holidays/holidays)
 
-A set of functions to deal with holidays in Ruby.
-
-Extends Ruby's built-in Date class and supports custom holiday definition lists.
+Functionality to deal with holidays in Ruby.
```
Comment (matkoniecz) [inline README.md]: I think that in this case original was better. It clearly described what is included in this library. New version is shorter, but original was not so long to make shortening it necessary.

File: holidays.gemspec [hunk 2]
```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
   gem.licenses      = ['MIT']
   gem.required_ruby_version = '~> 2.2'
-  gem.add_development_dependency 'bundler'
-  gem.add_development_dependency 'rake', '~> 12.0'
+  gem.add_development_dependency 'bundler', '~> 1.16'
```
Comment (ttwo32) [inline holidays.gemspec]: Why bundler's version is fixed?

File: test/integration/test_any_holidays_during_work_week.rb [hunk 3]
```diff
@@ -0,0 +1,90 @@
+require File.expand_path(File.dirname(__FILE__)) + '/../test_helper'
```
Comment (ttwo32) [inline test/integration/test_any_holidays_during_work_week.rb]: This is a good solution.:)
Dividing test cases is easy for programmers to understand.

File: lib/holidays.rb [hunk 4]
```diff
@@ -19,13 +19,12 @@ module Holidays
...
   class << self
     def any_holidays_during_work_week?(date, *options)
-      days_to_monday = date.wday - 1
-      days_to_friday = 5 - date.wday
-      start_date = date - days_to_monday
-      end_date = date + days_to_friday
-      options += [:observed] unless options.include?(:no_observed)
-      options.delete(:no_observed)
-      between(start_date, end_date, options).empty?
+      monday = date - (date.wday - 1)
```
Comment (ttwo32) [inline lib/holidays.rb]: LGTM!!
It's easy for me to understand logic and purpose.

File: README.md (hunk 1 above)
Comment (ppeble) [reply README.md]: Thank you! I've taken this into account and pushed up a new commit (37a2575). I'm going to go ahead and merge to clear this off of my plate but if you have additional feedback please add it here, I am more than happy to continue iterating so this is as clear as can be.

File: holidays.gemspec (hunk 2 above)
Comment (ppeble) [reply holidays.gemspec]: The gem build was giving me a warning telling me that it wasn't 'safe' to leave a version off of `bundler`. I added it in mainly to make that warning going away. 😄

Comment (ttwo32) [reply holidays.gemspec]: Oh, i see.:)
Please go ahead :rocket:

**Pull request merged**